runtime/include/chplsys.h

```diff
@@ -36,4 +36,9 @@ int chpl_getNumLogicalCpus(chpl_bool accessible_only);
 //
 c_string chpl_nodeName(void);
 
+//
+// Returns the value of a CHPL_RT_* environment variable, with default.
+//
+chpl_bool chpl_env_getBool(const char*, chpl_bool);
```

Comment:
Not sure if it should be done for this commit or not, but I think this function should be moved into a new header/source that only contains env var getters (and maybe setters?) 

I think we're going to want a bool, string, int, and B/KB/MB/GB versions as well and it seems like a good idea to have them all together in env var specific file.